landbosse/model/ErectionCost.py

```diff
@@ -160,10 +160,10 @@ def run_module(self):
                 project_id=self.project_name,
                 total_or_turbine=True
             )
-            return 0
-        except Exception:
+            return 0, 0 # Module ran successfully
```

Comment:
This is a bit of code cleanup I found in ErectionsCost. It makes exception handling consistent across all the modules.